templates/features.html

```diff
@@ -94,6 +94,12 @@
   search.value = location.hash.substr(1);
 }
 
+chromeMetadata.addEventListener('query-changed', function(e) {
+  search.value = '=' + e.detail.version;
+  var event = new Event('input');
```

Comment:
this does feel a bit hacky to me. The `input` listener doesn't up the change to `search.value`?

If no, just call `featureList.filter(search.value);` instead of firing the event. I don't see much value in arbitrating through the search box.